### PR TITLE
chore(misc): Various stuff

### PR DESCRIPTION
**Pull request by InfamousVague**
### What this PR does 📖
 Adds feature flipper:
image
image
image
image

Correctly aligns reactions in messages:
image

Moves reactions into context menu: 
image

### CODE REVIEW

**Comment by phillsatellite** [comment]
Veryyyyy minor ui thing I found, it looks like each reaction has a tooltip if you hover over it? Was this intended? Either way awesome PR and worth getting in sooner rather than later for upcoming small release 

Screenshot 2023-07-20 at 2 28 25 PM

**Review by phillsatellite — APPROVED** [review]

File: ui/src/components/chat/compose/messages.rs [hunk 1]
```diff
@@ -798,49 +796,47 @@ fn render_message<'a>(cx: Scope<'a, MessageProps<'a>>) -> Element<'a> {
...
-                                },
-                                "{reaction}"
-                            }
-                        )
-                    })
-                },
-                script { focus_script },
-            )
-        }),
+        // (*reacting_to.current() == Some(*msg_uuid)).then(|| {
```
Comment (lgmarchi) [inline ui/src/components/chat/compose/messages.rs:806]: Is it not better to remove all this code?

**Comment by phillsatellite** [comment]
Ready to merge when linter is fixed 💥

**Review by WanderingHogan — APPROVED** [review]

File: ui/src/components/chat/compose/messages.rs (hunk 1 above)
Comment (WanderingHogan) [reply ui/src/components/chat/compose/messages.rs:806]: gonna leave this in for now and revisit later

**Pull request merged**